api4/cloud_test.go

```diff
@@ -649,6 +649,58 @@ func TestGetCloudProducts(t *testing.T) {
 	})
 }
 
+func Test_GetExpandStatsForSubscription(t *testing.T) {
+	isExpandable := &model.SubscriptionExpandStatus{
+		IsExpandable: true,
+	}
+
+	licenseId := "licenseID"
+
+	t.Run("NON Admin users are UNABLE to request expand stats for the subscription", func(t *testing.T) {
+		th := Setup(t).InitBasic()
+		defer th.TearDown()
+
+		th.Client.Login(th.BasicUser.Email, th.BasicUser.Password)
+
+		cloud := mocks.CloudInterface{}
+
+		cloud.Mock.On("GetLicenseExpandStatus", mock.Anything).Return(isExpandable, nil)
+
+		cloudImpl := th.App.Srv().Cloud
+		defer func() {
+			th.App.Srv().Cloud = cloudImpl
+		}()
+		th.App.Srv().Cloud = &cloud
+
+		subscriptionExpandable, r, err := th.Client.GetExpandStats(licenseId)
```

Comment:
Looks like the test name up above and this modal method could use a rename to `Status`